docs/tutorial.md

```diff
@@ -11,7 +11,9 @@ This text assumes that the reader is familiar with Git.
 If you haven't already, make sure you
 [install and configure Jujutsu](install-and-setup.md).
 
-## Cloning a Git repo
+## Cloning a Git repository
+
+>  **Hint:** Most identifiers used in this tutorial will be different when you try this at home!
```

Comment:
I reformatted the file in https://github.com/martinvonz/jj/commit/2f8917145a0077db2c4770b3de71836942941b77

It's partially an automatted reformat and partially manually where sentences would be split awkwardly otherwise.